docs/dataset.rst

```diff
@@ -81,3 +81,22 @@ DatasetCifar
 
 .. autoapiclass:: DatasetCifar
     :members:
+
+
+NLP
+~~~~~~~~~~~~~~~~
+
+.. automodule:: mlbench_core.dataset.nlp
+.. currentmodule:: mlbench_core.dataset.nlp
+
+pytorch
++++++++
+
+.. automodule:: mlbench_core.dataset.nlp.pytorch
```

Comment:
Can you change this file's directives to use `autoapi`, like in `models.rst` ?

Reply:
good catch, I think this snuck back in when I rebased on develop :+1: